RPIPs/RPIP-53.md

```diff
@@ -183,5 +183,136 @@ v10 has been implemented into a Smart Node branch [here](https://github.com/rock
 Aside from variable names, the actual functional changes are very small. It just checks to see if a node has an RPL withdrawal address set, and if so, it directs the RPL earnings to the RPL withdrawal address and the ETH earnings to the primary withdrawal address instead.
 
 
+## Appendix A: Setting the RPL Withdrawal Address to a Registered Node
+
+The results of setting your RPL withdrawal address to a registered node address can be somewhat confusing, so the scenarios are spelled out explicitly here to dispel any uncertainty.
+
+
+### Scenario 1
+
+In this scenario, we have the following setup:
+
+```
+Node A: primary A, RPL B
+Node B: primary C, RPL not set
+```
+
+After a rewards event is triggered, the following claimers will be produced in the tree:
+
+```
+Entry A (A's ETH, A's RPL)
+
+Entry B (B's ETH, B's RPL)
+```
+
+- Eligible claimers for Entry A and Entry B are both determined at the time of claiming.
+
+Note that if this is the scenario at the time of claiming Entry A, then only B can claim Entry A. However, the rewards will go to A and C. B itself won't earn anything and it will have to pay for gas, so this situation only works trustlessly when the owner of B also owns C.
```

Comment:
Why does C get node A's rewards?

Reply:
Looks like that's a mistake, thanks for the catch. Updated to reflect that B will get the RPL here.